src/main/java/hudson/plugins/logparser/LogParserParser.java

```diff
@@ -69,8 +70,12 @@ public LogParserParser(final FilePath parsingRulesFile,
      * lists of links to these errors/warnings/info messages respectively :
      * errorLinks.html, warningLinks.html, infoLinks.html
      */
-    public LogParserResult parseLog(final AbstractBuild build)
-            throws IOException, InterruptedException {
+    @Deprecated
+    public LogParserResult parseLog(final AbstractBuild build) throws IOException, InterruptedException {
+        return this.parseLog((Run<?, ?>) build);
+    }
+
+    public LogParserResult parseLog(final Run<?, ?> build) throws IOException, InterruptedException {

```

Comment:
@oleg-nenashev I thought the same but I'm fairly sure @jglick said it was bin-incompatible. I'm not sure how though as I think I created a pair of test projects at the time (out of curiosity) and it seemed to work fine.